internal/controller/coralogix/rulegroup_controller.go

```diff
@@ -156,6 +148,10 @@ func (r *RuleGroupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 			id := createRuleGroupResp.GetRuleGroup().GetId().GetValue()
 			ruleGroupCRD.Status = coralogixv1alpha1.RuleGroupStatus{ID: &id}
 			if err := r.Status().Update(ctx, ruleGroupCRD); err != nil {
+				if err := r.deleteRemoteRuleGroup(ctx, log, ruleGroupCRD.Status.ID); err != nil {
+					log.Error(err, "Error on deleting RecordingRuleGroupSet after status update error", "Name", ruleGroupCRD.Name, "Namespace", ruleGroupCRD.Namespace)
```

Comment:
`log.V(1).Error` ?

Reply:
Seems like this big Reconcile func uses `log.Error` for error. We need to handle this logging differences in another PR.